### PR TITLE
[ETH-FLOW-V2] Update UI for EthFlow orders

### PR DESCRIPTION
**Pull request by W3stside**
# Summary

Adds UI changes necessary for making real on-chain EthFlow orders

- remove `SwapWithWrappedToken` state for `EthFlowOrder` state 
- update `serializeToken` to work with native currency

## QA - nothing to test here, wait for incoming PRs
- ETH <> token swaps will show the correct button but don't expect the UX to work

### CODE REVIEW

File: src/custom/state/orders/mocks.ts [hunk 1]
```diff
@@ -41,6 +43,7 @@ const generateOrderId = (ind: number) => {
 }
 
 export const generateOrder = ({ owner, sellToken, buyToken }: GenerateOrderParams): Order => {
+  console.error('🚀 ~ file: mocks.ts ~ line 45 ~ generateOrder ~ owner', owner)
```
Comment (W3stside) [inline src/custom/state/orders/mocks.ts:46]:
```suggestion
```

File: src/custom/state/orders/mocks.ts [hunk 2]
```diff
@@ -75,3 +77,59 @@ export const generateOrder = ({ owner, sellToken, buyToken }: GenerateOrderParam
...
+  const order = generateOrder(props)
+  return {
+    ...order,
+    inputToken: serializeToken(order.inputToken),
+    outputToken: serializeToken(order.outputToken),
+  }
+}
+
+// can use to test toast notification and state updates for different order types
+export const mockOrderDispatches = {
```
Comment (W3stside) [inline src/custom/state/orders/mocks.ts:91]: for testing toasts

File: src/custom/state/user/hooks/index.ts [hunk 3]
```diff
@@ -71,3 +73,15 @@ export function useInitFavouriteTokens(): void {
     }
   }, [chainId, dispatch])
 }
+
+export function serializeToken(token: Currency): SerializedToken {
```
Comment (shoom3301) [inline src/custom/state/user/hooks/index.ts:77]: Let's move it to `cow-react/utils/tokens.ts`

**Review by josojo — APPROVED** [review]

File: src/custom/state/orders/mocks.ts [hunk 4]
```diff
@@ -75,3 +77,59 @@ export const generateOrder = ({ owner, sellToken, buyToken }: GenerateOrderParam
     apiAdditionalInfo: undefined,
   }
 }
+
+export function generateSerializedOrder(props: GenerateOrderParams): SerializedOrder {
```
Comment (shoom3301) [inline src/custom/state/orders/mocks.ts:81]: If it's not an override then it should be in `cow-react folder`

**Review by shoom3301 — APPROVED** [review]

**Review by alfetopito — APPROVED** [review]

**Comment by W3stside** [comment]
making a note in the base PR to move all these files at the end to not force me to back rebase all the waterfalls rn

**Pull request merged**